python/Ganga/new_tests/GPI/Bugs/Savannah10064.py

```diff
@@ -5,7 +5,7 @@
 
 class Savannah10064(GangaUnitTest):
 
-    def Savannah10064(self):
+    def test_Savannah10064(self):
```

Comment:
Upon more though I'm happy to keep it how you've changed it. You're right that it's nice to be able to call `nosetests` simply without any complicated `--testmatch` setup.